src/AutoRest.CSharp/AutoRest.CSharp.csproj

```diff
@@ -41,6 +41,11 @@
     <Content Include="build\CodeGeneration.targets" PackagePath="build\Microsoft.Azure.AutoRest.CSharp.targets" />
     <Content Include="build\CodeGeneration.targets" PackagePath="buildMultiTargeting\Microsoft.Azure.AutoRest.CSharp.targets" />
   </ItemGroup>
+  
+  <ItemGroup>
+    <Content Include="../../node_modules/@azure-tools/cadl-csharp/**/*" PackagePath="tools\cadl-csharp" />
```

Comment:
I think this will cause typescript files to be built into the DLL.

Reply:
No, it puts js files into nuget package.